extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -370,16 +374,14 @@ private void applyBuildItems(Session session,
         String openshiftName = openshiftConfig.name.orElse(name);
         String knativeName = knativeConfig.name.orElse(name);
 
-        //TODO: This is needed until https://github.com/dekorateio/dekorate/issues/456 is resolved.
```

Comment:
This comment seems to have been removed from the code that adds `ApplyContainerImageDecorator`. Should it be re-added sicne the issue is still open or has the issue been side-stepped?